spec/in_paralell_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rspec'
 require_relative('../lib/in_parallel')
 include InParallel
-TMP_FILE = '/tmp/test_file.txt'
+TMP_FILE = 'tmp/test_file.txt'
```

Comment:
Instead of using a relative path here, why not use the builtin `Dir` class to create a [temporary file](http://ruby-doc.org/stdlib-2.3.1/libdoc/tmpdir/rdoc/Dir.html#method-c-tmpdir)? This way, you won't pollute the repo directory with test fixtures.